File: extract_pdf.js
const pdf = require('pdf-parse');
const fs = require('fs');
const buf = fs.readFileSync('src/assets/Manish Chavan Resume.pdf');
pdf(buf).then(data => {
  fs.writeFileSync('resume_out.txt', data.text);
  console.log(data.text);
}).catch(err => console.error(err.message));

File: extract_pdf.py
import sys
from PyPDF2 import PdfReader

try:
    reader = PdfReader('src/assets/Manish Chavan Resume.pdf')
    text = ""
    for page in reader.pages:
        text += page.extract_text() + "\n"
    with open('resume_out.txt', 'w', encoding='utf-8') as f:
        f.write(text)
    print("Success")
except Exception as e:
    print(f"Error: {e}")
